### ISSUE TITLE
Add new policies

### ISSUE THREAD

**Issue by HenryJia** [keras-rl/keras-rl#23]
Softmax policy and gaussian policy are some pretty common policies.

Softmax policy is where it's sampling using the softmax output probabilities.

Gaussian policy is for real value actions. So the network outputs the mean and stddev (or stddev is a fixed constant) and outputs are sampled from the gaussian distribution.

**Comment by matthiasplappert** [comment]
Softmax is already implemented for Q learning (as BoltzmannQPolicy).
